CONTRIBUTING.md

```diff
@@ -1,79 +1,5 @@
-# Contribution guidelines
+# Contributing to sbi
 
-By participating in the `sbi` community, all members are expected to comply with our [Code
-of Conduct](CODE_OF_CONDUCT.md). This ensures a positive and inclusive environment for
-everyone involved.
+The latest contributing guide is available in the repository at `docs/contribute.md`, or online at:
 
-## User experiences, bugs, and feature requests
-
-If you are using `sbi` to infer the parameters of a simulator, we would be delighted to
-know how it worked for you. If it didn't work according to plan, please open up an
-[issue](https://github.com/sbi-dev/sbi/issues) or
-[discussion](https://github.com/sbi-dev/sbi/discussions) and tell us more about your use
-case: the dimensionality of the input parameters and of the output, as well as the setup
-you used to run inference (i.e., number of simulations, number of rounds, etc.).
-
-To report bugs and suggest features (including better documentation), please equally
-head over to [issues on GitHub](https://github.com/sbi-dev/sbi/issues).
-
-## Code contributions
-
-Contributions to the `sbi` package are welcome! In general, we use pull requests to make
-changes to `sbi`. So, if you are planning to make a contribution, please fork, create a
-feature branch, and then make a Pull Request (PR) from your feature branch to the upstream `sbi`
-([details](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)).
-To give credit to contributors, we consider adding contributors who repeatedly and
-substantially contributed to `sbi` to the list of authors of the package at the end of
-every year. Additionally, we mention all contributors in the releases.
-
-### Development environment
-
-Clone [the repo](https://github.com/sbi-dev/sbi) and install all the dependencies via
-`pyproject.toml` using `pip install -e ".[dev]"` (the `-e` flag installs the package
-editable mode, and the `dev` flag installs development and testing dependencies).
-This requires at least Python 3.8.
-
-We use [`pre-commit`](https://pre-commit.com) to ensure proper formatting and perform
-linting (see below). Please install `pre-commit` locally using `pre-commit install`.
-
-### Style conventions and testing
-
-For docstrings and comments, we use [Google
-Style](http://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings).
-
-For code linting and formating, we use [`ruff`](https://docs.astral.sh/ruff/), which is
-installed alongside `sbi`.
-
-You can exclude slow tests and those which require a GPU with `pytest -m "not slow and not gpu"`.
```

Comment:
Thanks for rebasing but git cannot recognize that you moved the content from here to the other file. This would only be recognized if you would have renamed this file.

This means that you're deleting changes, for example this part.